web/client/themes/default/less/common.less

```diff
@@ -76,6 +76,23 @@
             .background-color-var(@theme-vars[main-bg]);
         }
     }
+    .form-group-flex {
```

Comment:
this file is missing in original PR is because stable branch had some missing code due to other PRs not being backported?